components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
+         {{/if}}
+     "
+     {{{ attributes }}}
+>
+    <div class="
+             gallery__stage
+             {{ stage.class }}
+             {{#if typeHorizontal }}
+                 gallery__stage--horizontal
+             {{else}}
+                 gallery__stage--vertical
+             {{/if}}
+         "
+    >
+        <div class="gallery__arr gallery__arr--prev">
+        </div>
+        <div class="gallery__shaft">
```

Comment:
this class don't have any styles and is it required to add next level of DOM here?

Reply:
In fotorama it's anyway (even more elements...) just wanted to build on skeleton of fotorama in case we could incorporate fotorama fully here. I think for "placeholder" it's no big deal but will remove it.